lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -359,6 +359,11 @@ def configure(conf)
         @dropped_entries_count = registry.counter(
           :stackdriver_dropped_entries_count,
           'A number of log entries dropped by the Stackdriver output plugin')
+        @log_entry_retry_count = registry.counter(
```

Comment:
I would call it `@retried_entries_count` for consistency... But what you have is fine too.